pkg/client/integration_test.go

```diff
@@ -3,6 +3,9 @@ package client_test
 import (
 	"bytes"
 	"context"
+	"crypto/hkdf"
```

Comment:
Package is only available from go 1.24 onward. We support the latest 2 versions of go, that means 1.23.

Reply:
Apologies for the delay - I've updated the PR to use HMAC directly rather than HKDF :)